pkg/kube/pod.go

```diff
@@ -31,13 +33,16 @@ import (
 
 	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
 	"github.com/kanisterio/kanister/pkg/consts"
+	"github.com/kanisterio/kanister/pkg/field"
 	"github.com/kanisterio/kanister/pkg/log"
 	"github.com/kanisterio/kanister/pkg/poll"
 )
 
 const (
-	// podReadyWaitTimeout is the time to wait for pod to be ready
-	podReadyWaitTimeout  = 15 * time.Minute
+	// defaultPodReadyWaitTimeout is the time to wait for pod to be ready
+	defaultPodReadyWaitTimeout = 15 * time.Minute
+	// podStartTimeoutEnv is the env var to get pod startup wait timeout
+	podStartTimeoutEnv   = "KANISTER_POD_START_TIMEOUT"
```

Comment:
Lets name this `podReadyWaitTimeoutEnv` so that it sounds similar to the var that represents the default value  `defaultPodReadyWaitTimeout` ?  This also means we may want to rename the env to KANISTER_POD_READY_WAIT_TIMEOUT .